pkgs/development/python-modules/bottombar/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flit-core
+}:
+
+buildPythonPackage rec {
+  pname = "bottombar";
+  version = "1.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "evalf";
+    repo = "bottombar";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-/3m34HcYmmEf92H3938dYV1Q6k44KaCb9TDx9nDNPnM=";
+  };
+
+  propagatedBuildInputs = [
+    flit-core
+  ];
+
+  # The package only has some "interactive" tests where a user must check for
+  # the correct output and hit enter after every check
```

Comment:
In that case you should set `doCheck = false`.

Reply:
Don't remove the `pythonImportsCheck`. It's all we have, when t here are no actual tests.

Reply:
i see